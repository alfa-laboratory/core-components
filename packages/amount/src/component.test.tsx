import React from 'react';
import { render } from '@testing-library/react';

import { Amount } from '.';

describe('Amount', () => {
    it('should match snapshots for base and Pure components', () => {
        const { container } = render(
            <React.Fragment>
                <Amount value={100} currency='RUR' minority={100} />
                <Amount.Pure value={100} currency='RUR' minority={100} />
            </React.Fragment>,
        );
        expect(container).toMatchSnapshot();
    });
});
