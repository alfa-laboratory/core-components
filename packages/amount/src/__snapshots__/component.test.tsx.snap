// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amount should match snapshots for base and Pure components 1`] = `
<div>
  <span
    class="component"
  >
    1
    <span
      class="minorPartAndCurrency"
    >
      
      
       
      ₽
    </span>
  </span>
  <span
    class="component"
  >
    1 ₽
  </span>
</div>
`;
